### ISSUE TITLE
[Feature Request] Add option to sweep whole wallet balance (similar to sweep_all)

### ISSUE THREAD

**Issue by dEBRUYNE-1** [mymonero/mymonero-app-js#92]
It would be convenient for an user to be able to "sweep" the whole balance of the wallet, where the fee is subtracted from the total amount and the remainder is sent to the destination chosen by the user. 

Perhaps it would be prudent to add a warning that sweeping the whole balance is possibly detrimental to the user's privacy.

**Comment by paulshapiro** [comment]
I've just talked with vtnerd about this, and I was able to confirm that the client should be able to effectively do the same thing as a sweep all by moving (sending) all outputs which are not spent, to another, desired address, by way of a normal `submit_raw_tx`. So I'm removing the 'may need server' tag.

However, wallet2.cpp does do some special output selection by histogram in order to mitigate linkability. This could be reproduced in Javascript but my inclination is to say that we should wait until we can integrate with the official baseline wallet2.cpp, for many reasons. So I'm putting this one on ice for now. :)
